tasks/internal/process-file-updates-task/process-file-updates-task.yaml

```diff
@@ -251,35 +251,33 @@ spec:
         fi
 
         echo -e "\n*** START LOCAL CHANGES ***\n"
-        echo -e "\n*** Result from git diff ***\n"
```

Comment:
Why remove this?

Reply:
ok, now we no longer do `git diff`, because it would never show anything (because we do `git add` now), correct?